### PR TITLE
Add instructions for using napari hub shield to FAQ

### PR DESCRIPTION
**Pull request by DragaDoncila**
Add instructions for embedding shield into markdown (linking back to the hub) or just accessing the SVG.

Not sure if this is the best place to put this, but it made sense to me to put it under "Customizing your plugin's listing" - @neuromusic thoughts and/or concerns?

I'll also open a PR on the cookiecutter reop to add it to our README there, and once we release next week, advertise in the community meeting and on Zulip.

### CODE REVIEW

File: frontend/src/pages/faq.mdx [hunk 1]
```diff
@@ -147,6 +147,22 @@ to see what metadata we will present and where we source it from.
 
 </Accordion>
 
+<Accordion
+  title="How do I add a shield/badge for the hub to my repository?"
```
Comment (neuromusic) [inline frontend/src/pages/faq.mdx:151]:
```suggestion
  title="How do I add a shield/badge for the napari hub to my repository?"
```

File: frontend/src/pages/faq.mdx [hunk 2]
```diff
@@ -147,6 +147,22 @@ to see what metadata we will present and where we source it from.
...
+  variant="faq"
+>
+
+The napari hub supports [Shields.io](https://shields.io/) shields for your repository linking to your plugin’s 
+listing page. If you want to add a shield to your README or other markdown files, 
+you can copy paste this snippet:
```
Comment (neuromusic) [inline frontend/src/pages/faq.mdx:157]:
```suggestion
you can copy & paste the snippet, replacing both instances of `<your-plugin-name-here>` with your plugin's package name:
```

**Review by neuromusic — CHANGES_REQUESTED** [review]
Couple of changes, but otherwise LGTM!

**Comment by DragaDoncila** [comment]
@neuromusic made those changes now (and fixed the tests)

**Review by neuromusic — APPROVED** [review]
🎉

**Pull request merged**